server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/GroupWsRef.java

```diff
@@ -21,21 +21,23 @@
 
 import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
+import org.sonar.api.security.DefaultGroups;
 
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
+import static org.sonar.server.ws.WsUtils.checkRequest;
 
 @Immutable
-public class GroupRef {
+public class GroupWsRef {
```

Comment:
`WsGroupRef` and `GroupId`, `GroupIdOrAnyone`.... a little documentation on these classes wouldn't hurt